src/sentry/static/sentry/app/components/externalLink.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../less/components/externalLink.less';
+
+/**
+ * Use this component when creating external links.
+ *
+ * By default will open in a new tab with `rel="noreferrer noopener"` to guard against
+ * `target="_blank"` vulnerabilities
+ */
+export default function ExternalLink({href, ...otherProps}) {
+  return <a {...otherProps} href={href} />;
+}
+
+ExternalLink.propTypes = {
+  href: PropTypes.string.isRequired,
+  target: PropTypes.string,
+  rel: PropTypes.string
+};
+
+// Should we allow these to be overridden?
+ExternalLink.defaultProps = {
+  target: '_blank',
```

Comment:
when would someone overwrite these? maybe this is premature and they should just be hardcoded so they don't show up in storybook and add cognitive overhead?

Reply:
It's possible you could have a link that opens to sentry docs or something and you don't want it to be in a new tab. I think it's fine to have these props in storybook since they have defaults.